<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenuButtons</class>
 <widget class="QDialog" name="AdminMenuButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
    background-color: #DDDDDD;
    border: 1px solid #1e1e1e;
    border-radius: 1px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="show_images">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>Show Images</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="update_price">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>Update Price</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_dia">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>Add Diamond</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="jewelry_menu_button">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>Jewelry Menu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="orderBookUsersPushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>OrderBook Users</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="orderBookRequestPushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>OrderBook Request</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="show_users">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #EEEEEE;
    color: #111111; 
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 20px;
    min-width: 80px;
    text-align: center;
    border: 1px solid #CCCCCC;
}

/* Hover Effect */
QPushButton:hover {
    background-color: #222222;
    color: #FFFFFF;
    border: 1px solid #1e1e1e;
}

/* Pressed Effect */
QPushButton:pressed {
    background-color: #444444;
    color: #FFFFFF;
    border: 1px solid #B57D2A;
}

/* Disabled State */
QPushButton:disabled {
    background-color: #DDDDDD;
    color: #888888;
    border: 1px solid #CCCCCC;
}
</string>
     </property>
     <property name="text">
      <string>Show Users</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="logout">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #D62828;       /* Deep red */
    color: #FFFFFF;
    font-size: 12px;
    font-weight: bold;
    border-radius: 4px;
    padding: 8px 22px;
    min-width: 100px;
    text-align: center;
    border: 1px solid #8B0000;       /* Dark border for definition */
}

QPushButton:hover {
    background-color: #B71C1C;       /* Darker red on hover */
    border: 1px solid #600000;
}

QPushButton:pressed {
    background-color: #7F1D1D;       /* Very dark red on press */
    border: 1px solid #4A0E0E;
}

QPushButton:disabled {
    background-color: #E0E0E0;
    color: #9E9E9E;
    border: 1px solid #BDBDBD;
}
</string>
     </property>
     <property name="text">
      <string>‚èª      LOGOUT</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
